docs/multichain.md

```diff
@@ -2,20 +2,190 @@
+blockchain without accessing the network if there are constant header updates
+from whoever submits valid ones.
+
+Thanks to this simple but powerful property, it is possible to embed a light
+client as a smart contract. This enables **trustless message delivery between
+two blockchains**, which is the key to multichain interoperability.
+
+Suppose there are two chains called A and B. If the A chain has a light client
+of the B Chain deployed on it, whatever is finalized in the B chain can be
+instantly verified in the A chain. Other contracts deployed on the A chain would
+interact with the B chain since the light client provides data from the B data
+after verification. This communication can trivially become two-way if the B
+chain has a light client of the A chain as well.
+
+Cosmos IBC is an example of a protocol based on this idea. IBC defines multiple
+layers over the core communication primitive (two chains embedding light clients
+of each other) to establish high-level communication such as asset transfer.
+Various chains built with Cosmos SDK (a.k.a. zones) are all standalone layer-1
+networks, but they form an organic multichain structure using IBC.
+
+## Settlement Chain
+
+Simperby is a standalone blockchain that has its own consensus and distributed
+file system. However, there must be working applications that interact with the
+chain so that the organization can finalize something actually meaningful. Of
+course, the most important applications would be existing other chains that the
+organization deploys their applications on. From Simperby's view, they are
+called **settlement chains**. Any blockchain that has a smart contract platform
+can be a settlement chain once the organization pays some gas and deploys their
+light client on it.
+
+Simperby's scalability on settlement chain integration is quite impressive; the
+code for the light client contract will be almost the same for every chain
+except for a few virtual machine-specific behaviors and the host interface. This
+enables **scalable development of light clients over various settlement
+chains**, which eventually provides powerful multichain interoperability for the
+DAO that uses Simperby.
+
+Also, considering that a Simperby chain has no contracts or business logic but
+is only capable of recording explicitly approved data, there is nothing to
+execute automatically from the Simperby side. That is, the multichain
+communication system **doesn't have to be two-way**. Things happen (finalized)
+in the settlement chain might be delivered to the Simperby chain in the same way
+(embedding a light client) but that's just pointless; there's nothing to
+programmatically respond to such events. Instead, members of the chain will
+manually check the result from the settlement using whatever method they want
+and take it into account if it matters in deciding the next agenda in the
+Simperby chain. It could be a standalone light client, some centralized explorer
```

Comment:
```suggestion
and take it into account if it matters in making a decision on the next agenda. It could be a standalone light client, some centralized explorer
```